lib/DocumentInspectorFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager;
+
+/**
+ * Document inspector factory for the default DocumentInspector.
+ */
+class DocumentInspectorFactory implements DocumentInspectorFactoryInterface
```

Comment:
We did, but at this stage it requires more effort than it is worth, and
then I am not sure if it is the best thing to do. Would rather wait
before removing this, if at all.

On Wed, Apr 27, 2016 at 07:07:36AM -0700, Daniel Rotter wrote:
>    In [1]lib/DocumentInspectorFactory.php:
> 
>  > +
>  > +/*
>  > + * This file is part of Sulu.
>  > + *
>  > + * (c) MASSIVE ART WebServices GmbH
>  > + *
>  > + * This source file is subject to the MIT license that is bundled
>  > + * with this source code in the file LICENSE.
>  > + */
>  > +
>  > +namespace Sulu\Component\DocumentManager;
>  > +
>  > +/**
>  > + * Document inspector factory for the default DocumentInspector.
>  > + */
>  > +class DocumentInspectorFactory implements DocumentInspectorFactoryInterface
> 
>    Didn't we say we remove this class? I thought you've added the comment
>    somewhere, but I can't find it anymore... Shouldn't that have been merged
>    with the contexts (because I've already seen the contexts in here)?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/66#discussion_r61262941
>    2. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61262941
>    3. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61262941